primary/Tenkuu Danzai Skelter Heaven (primary).ass

```diff
@@ -161,110 +159,94 @@ Dialogue: 0,0:10:53.97,0:10:54.83,Default,,0,0,0,,And Midori!
+Dialogue: 0,0:12:46.75,0:12:50.50,Default,,0,0,0,,As the first generation of homunculi, \Nyou girls are the prototypes.
 Dialogue: 0,0:12:51.52,0:12:53.13,Default,,0,0,0,,That's enough pointless chitchat.
 Dialogue: 0,0:12:53.13,0:12:55.49,Default,,0,0,0,,You just need to follow orders.
 Dialogue: 0,0:12:55.49,0:12:56.79,Default,,0,0,0,,All right?
 Dialogue: 0,0:12:56.79,0:12:57.98,Default,,0,0,0,,Understood.
 Dialogue: 0,0:12:59.78,0:13:00.99,Default,,0,0,0,,That's good.
 Dialogue: 0,0:13:00.99,0:13:03.35,Default,,0,0,0,,Your orders are absolute.
-Dialogue: 0,0:13:03.35,0:13:06.46,Default,edit,0,0,0,,You're as dutiful as they come, aren't you?{like the impulse to question his orders would never enter her mind}
+Dialogue: 0,0:13:03.35,0:13:06.46,Default,edit,0,0,0,,You don't have an insubordinate \Nbone in your body, do you?{You're as dutiful as they come, aren't you?}{like the impulse to question his orders would never enter her mind}
 Dialogue: 0,0:13:07.01,0:13:07.89,Default,,0,0,0,,What's that?
 Dialogue: 0,0:13:19.44,0:13:21.78,Default,,0,0,0,,Rin, focus!
-Dialogue: 0,0:13:25.38,0:13:27.04,Default,,0,0,0,,I'm fine, I'm fine.
+Dialogue: 0,0:13:25.38,0:13:27.04,Default,,0,0,0,,I'm good! I'm good.
 Dialogue: 0,0:13:27.04,0:13:28.15,Default,,0,0,0,,I can still fight!
-Dialogue: 0,0:13:28.15,0:13:30.40,Default,,0,0,0,,Rin... no.
+Dialogue: 0,0:13:28.15,0:13:30.40,Default,,0,0,0,,Rin—No.
 Dialogue: 0,0:13:30.40,0:13:31.68,Default,,0,0,0,,All units, retreat.
 Dialogue: 0,0:13:33.16,0:13:35.17,Default,,0,0,0,,I'll finish it off.
-Dialogue: 0,0:13:35.17,0:13:38.77,Default,,0,0,0,,As long as we know its weakness,\NI can simply charge into it with my unit!{弱点さえ分かれば、こいつで突っ込むのもわけないさ}{pretty sure he means as a suicide attack, with the alternative in his mind being one of the girls doing the same thing}
-Dialogue: 0,0:13:38.77,0:13:40.73,Default,,0,0,0,,Funagai! Have you gone insane?
-Dialogue: 0,0:13:42.17,0:13:43.90,Default,,0,0,0,,I told myself it was just work.
-Dialogue: 0,0:13:45.61,0:13:48.71,Default,,0,0,0,,I was able to cope by convincing myself it was professional.
-Dialogue: 0,0:13:49.37,0:13:50.31,Default,,0,0,0,,But...
-Dialogue: 0,0:13:51.44,0:13:52.71,Default,,0,0,0,,those girls
-Dialogue: 0,0:13:53.21,0:13:54.40,Default,,0,0,0,,were smiling.
-Dialogue: 0,0:13:56.11,0:13:58.30,Default,,0,0,0,,That's why... That's why...
+Dialogue: 0,0:13:35.17,0:13:38.77,Default,,0,0,0,,We know its weak spot now, \Nso all I've gotta do is ram into it with my unit!{弱点さえ分かれば、こいつで突っ込むのもわけないさ}{pretty sure he means as a suicide attack, with the alternative in his mind being one of the girls doing the same thing}
+Dialogue: 0,0:13:38.77,0:13:40.73,Default,,0,0,0,,Funagai! Have you lost your mind?
+Dialogue: 0,0:13:42.17,0:13:43.90,Default,,0,0,0,,I told myself I was just doing my job.
+Dialogue: 0,0:13:45.61,0:13:48.71,Default,,0,0,0,,I coped with everything by convincing \Nmyself that was true.
```

Comment:
Seems a bit awkward to me to break it up there. I'd probably break before "convincing" or before "that", but the latter would be too imbalanced. I think you need an extra "that" after "myself", but then you end up with a double "that". Perhaps it needs rewriting:
```suggestion
Dialogue: 0,0:13:45.61,0:13:48.71,Default,,0,0,0,,I coped with everything by \Nconvincing myself of that.
```